R/utils_render_rtf.R

```diff
@@ -230,29 +253,56 @@ rtf_file <- function(header = NULL,
     header <- rtf_header(fonttbl, colortbl)
   }
 
-  # Include RTF code to express page numbering if `page_numbering != "none"`
-  if (page_numbering != "none") {
-
-    document <-
-      paste(
-        document,
-        "\n\n",
-        rtf_raw(
-          "{",
-          if (page_numbering == "footer") "\\footer" else "\\header",
-          "\\qr{\n{\\field{\\*\\fldinst {PAGE}}{\\fldrslt {Refresh >F9<}}}}\\par}"
-        )
+  # Create the `page_information` statements if not provided
+  if (is.null(page_information)) {
+
+    paper_width_val <- getOption(paste0("gt.rtf_paper_width_", page_orientation))
+    paper_height_val <- getOption(paste0("gt.rtf_paper_height_", page_orientation))
+
+    page_information <-
+      rtf_raw(
+        paste0("\\paperw", paper_width_val),
+        paste0("\\paperh", paper_height_val),
+        "\\widowctrl\\ftnbj\\fet0\\sectd\\linex0\n",
+        "\\lndscpsxn\n",
+        paste0("\\margl", getOption("gt.rtf_page_margin_left")),
+        paste0("\\margr", getOption("gt.rtf_page_margin_right")),
+        paste0("\\margt", getOption("gt.rtf_page_margin_top")),
+        paste0("\\margb", getOption("gt.rtf_page_margin_bottom")),
+        "\n",
+        paste0("\\headery", getOption("gt.rtf_page_header_height")),
+        paste0("\\footery", getOption("gt.rtf_page_footer_height")),
```

Comment:
Move these options to `tab_options`

Reply:
Rename to non-RTF, accept arbitrary absolute units

Reply:
Now done.